src/Resources/views/autocomplete/templates/item.html.twig

```diff
@@ -0,0 +1,15 @@
+<div class="aa-ItemWrapper">
```

Comment:
Shouldn't the item be clickable to be able to follow to the result? If it's not, the whole feature is quite useless 😅 

https://github.com/user-attachments/assets/61383fd2-d6ab-4d40-8671-8169851e6e7c

Reply:
Yes, it should and it is on my computer 😂  I will figure it out!

Reply:
Aaah, didn't notice you were clicking :D It works with the keyword. I will fix it! Thank you!